ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly-x86-64.yaml

```diff
@@ -54,7 +54,9 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: redhat-chaos-hog-scenarios
+    test:
+    - chain: redhat-chaos-krkn-hub-tests
```

Comment:
think we can run both these tests in above test run and keep this as the hog scenario run

Reply:
Done!